package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "raweb",
+    "name": "html",
```

Comment:
Interesting, where does `html` come from?

Reply:
Odd I did not check the package-lock.json diff -- this seems to have been autogenerated when running `sail npm install`.

Reply:
Figured this out - it's the default value because package.json does not have a name key (it must have at one point and was removed). I am pushing a fix.

Reply:
Ah, yes. I think that's fine to not have a name in there

Reply:
I went ahead and added it, as someone else will no doubt encounter this in the future when any new dependency is added to the package.json file.

Reply:
I'll probably remove it again in a year because i find it superfluous 😅 
I'll try to remember why it was added again.